instrumentation_test.go

```diff
@@ -172,6 +172,14 @@ func TestWithResourceAttributes(t *testing.T) {
 	})
 }
 
+func TestWithLogLevel(t *testing.T) {
+	c, err := newInstConfig(context.Background(), []InstrumentationOption{WithLogLevel("error")})
```

Comment:
Thanks for adding these tests! Do you mind extending the testing to cover what happens if invalid options are specified or the configuration doesn't contain logging level?

This is related to my earlier comment that parsing of the log level should be done here, so that invalid command line input is detected.

Reply:
Created the test to cover the validation of a wrong logging level